_style/method-invocation.md

```diff
@@ -92,35 +92,28 @@ gray area is short, operator-like methods like `max`, especially if commutative:
     // fairly common
     a max b
 
-Symbolic methods which take more than one parameter (they do exist!)
-may still be invoked using infix notation, delimited by spaces:
+Symbolic methods which take more than one parameter are discouraged.
+When they exist, they may still be invoked using infix notation, delimited by spaces:
 
     foo ** (bar, baz)
 
 Such methods are fairly rare, however, and should normally be avoided during API
-design. For example, the use of the `/:` and `:\` methods should be avoided in
+design. For example, the use of the (now deprecated) `/:` and `:\` methods should be avoided in
 preference to their better-known names, `foldLeft` and `foldRight`.
 
 ### Higher-Order Functions
 
-As noted, methods which take functions as parameters (such as `map` or
-`foreach`) should be invoked using infix notation. It is also *possible* to
-invoke such methods in the following way:
+Invoking higher-order functions may use parens or braces, but in
+either case, use dot notation and omit any space after the method name:
 
-    // wrong!
-    names.map { _.toUpperCase }
-
-This style is *not* the accepted standard! The reason to avoid this style is for
-situations where more than one invocation must be chained together:
+    names.map(_.toUpperCase)
```

Comment:
Good to remove this as this was the most problematic part. Suggestions:
* Scala 3 is more liberal with allowing parenthesis where scala 2 requires braces - perhaps this should be mentioned here?
* In places where there are multiple lines in the lambda, perhaps we want to include an example in the style guide of how lines are broken after the rocket `=>` etc., as people reading this standard might wonder how that could/should look.

Reply:
Yeah, but I'm going to consider it out of scope for this PR.